core/pom.xml

```diff
@@ -101,6 +101,14 @@
             <groupId>org.apache.shiro</groupId>
             <artifactId>shiro-crypto-hash</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shiro</groupId>
+            <artifactId>shiro-crypto-support-hashes-argon2</artifactId>
```

Comment:
nit: he groupId could be `org.apache.shiro.hashes` and artifactId: `shiro-crypto-argon2`
Or, `org.apache.shiro.crypto:shiro-hashes-argon2`